classes/ObjectModel.php

```diff
@@ -385,6 +391,10 @@ public function getFieldsLang()
      */
     public function getAssociatedLanguage(): Language
     {
+        static $language = null;
```

Comment:
I am not convinced by this static usage, from what I understood, shouldn't you use some Cache and/or lazy loading system ?

Reply:
You're right. The unused attribute was intended for this.